src/components/Gallery.tsx

```diff
@@ -1,15 +1,33 @@
 'use client'
-import React from 'react'
 import { Swiper, SwiperSlide } from 'swiper/react'
 import 'swiper/css'
+import 'swiper/css/pagination'
+import 'swiper/css/navigation'
+import {
+  Pagination,
+  Navigation,
+  Autoplay,
+  EffectCreative,
+} from 'swiper/modules'
 
-import { Autoplay, EffectCreative } from 'swiper/modules'
-
-export default function Gallery() {
+export default function Gallery({
+  pictures,
+  pagination,
+  loop,
+  autoHeight,
+}: {
+  pictures: string[]
+  pagination: boolean
+  loop: boolean
+  autoHeight: boolean
+}) {
   return (
     <div style={{ width: '100%' }}>
       <Swiper
         effect={'creative'}
+        pagination={pagination}
+        loop={loop}
```

Comment:
I think we can just hardcode this

```suggestion
        loop={true}
```